tests/test_equilibrium.py

```diff
@@ -292,20 +297,28 @@ def test_resolution():
 @pytest.mark.unit
 def test_equilibrium_from_near_axis():
     """Test loading a solution from pyQSC/pyQIC."""
-    qsc_path = "./tests/inputs/qsc_r2section5.5.pkl"
-    file = open(qsc_path, "rb")
-    na = pickle.load(file)
-    file.close()
+    from qic import Qic
+
+    na = Qic.from_paper("r2 section 5.2")
 
     r = 1e-2
     eq = Equilibrium.from_near_axis(na, r=r, M=8, N=8)
     grid = LinearGrid(M=eq.M_grid, N=eq.N_grid, NFP=eq.NFP, sym=eq.sym)
     data = eq.compute("|B|", grid=grid)
 
+    # get the sin/cos modes (we now the eq is symmetric)
```

Comment:
the eq is symmetric? I thought the test used an asymmetric eq, which I'd prefer to test we do it right for asymmetric NAE as well (even if the r2 section 5.5 asymmetry does not show up in the axis).